main.py

```diff
@@ -0,0 +1,44 @@
+from transformers import BeitFeatureExtractor, BeitForImageClassification
+import streamlit as st
+from PIL import Image
+
+
+def predict_classes(image):
+    feature_extractor = BeitFeatureExtractor.from_pretrained('microsoft/beit-base-patch16-224-pt22k-ft22k')
+    model = BeitForImageClassification.from_pretrained('microsoft/beit-base-patch16-224-pt22k-ft22k')
```

Comment:
This will reload the large model on each prediction. Try to avoid this in future work.